setup.py

```diff
@@ -104,6 +104,9 @@ def get_long_description():
 
 
 if __name__ == '__main__':
+    # Force "make develop" inside READTHEDOCS environment
+    if os.environ.get("READTHEDOCS") and "install" in sys.argv:
+        os.system("make develop")
```

Comment:
OK, fair enough, you're honestly calling a hack, which is always temporary, so it's acceptable by hacker standards! :)